README.md

```diff
@@ -1,12 +1,12 @@
 # README -  oVirt Engine Disaster Recovery #
 
-This is a web application, that uses Java + Ovirt API + PostgreSQL, to perform the task needed to recover from a Disaster Recovery. 
+This is a web application that uses Java, oVirt API and PostgreSQL to perform the task needed to for oVirt's Disaster Recovery. 
```

Comment:
PostgreSQL. This application currently needs to update oVirt Engine's Postgres database to edit the connection info.

Looking forward, we need to see if this can be done via the API, and if not - fix the API to do so so we can eliminate the direct database access.